### ISSUE TITLE
Window border broken

### ISSUE THREAD

**Issue by n0x-io** [vinceliuice/WhiteSur-kde#23]
Hello! Thank you so much for this amazing theme!

Sadly I am running into some issues regarding the border of windows:
![grafik](https://user-images.githubusercontent.com/16123199/90977436-d5c23480-e545-11ea-9ecc-bbe6ef57cb48.png)

There is a big border around the window and I have no idea how to get rid of it. 
I chose the WhiteSur-dark Window Decoration and left the window border size to default.
If I try for example setting the border size to non I still have a border around the window:
![grafik](https://user-images.githubusercontent.com/16123199/90977524-54b76d00-e546-11ea-8a5d-10d257841cf2.png)

And if I try any of the other scaling Window Decoration like for example WhiteSur-dark_x1.25 the title bar breaks:
![grafik](https://user-images.githubusercontent.com/16123199/90977590-d9a28680-e546-11ea-923d-269c8db02617.png)

Am I missing a setting or something else that could fix this weird issue?

Thank you in advance! If you need further information do not hesitate to ask :)

**Comment by vinceliuice** [comment]
You must using Nvidia driver

**Comment by n0x-io** [comment]
Yes I am using the official nvidia driver. Is it known for causing issues with the theme?

**Comment by vinceliuice** [comment]
Yes, all nvidia users have this issue, so I'm thinking about making a alt version for nvidia users now

**Comment by n0x-io** [comment]
Thanks for letting me know! I wasn't aware that this could have been caused by the nvidia driver since I never had any bigger issues with other themes.

**Comment by CodeForeverAndEver** [comment]
> Yes, all nvidia users have this issue, so I'm thinking about making a alt version for nvidia users now

@vinceliuice That would be awesome!

**Comment by silocoder** [comment]
Hi I use nvidia driver.

Some of the white sur application selection if you have borders (l/r/b) and click on it by accident when trying to resize of example, it brings the application below to the foreground. For now I have set it to no borders and only style that works is the Whitesharp. 

Thanks!

**Comment by nhactung** [comment]
I don't use either Nvidia or AMD and encounter this issue too...

**Comment by AzO95** [comment]
I have the same probems 
![image](https://user-images.githubusercontent.com/8530497/96252009-921e0100-0fb1-11eb-88ac-255787c98379.png)

**Comment by Kasumisty** [comment]
Both of my PCs are using NV GPUs. However, the theme works well on the one whose GPU is NV Geforce GT 730 while the other encounters the same problems as @n0x-io using NV Geforce RTX 2060 super.

**Comment by Hobr** [comment]
> Both of my PCs are using NV GPUs. However, the theme works well on the one whose GPU is NV Geforce GT 730 while the other encounters the same problems as @n0x-io using NV Geforce RTX 2060 super.

After updating the GPU driver,my GT 730 also works well now.It's obviously caused by NVIDIA's driver: )

**Comment by Kasumisty** [comment]
What version of you driver is it?

**Comment by MLDamazigh** [comment]
I have an integrated intel GPU (intel HD Graphics 2500) and i am facing the same problem, so the issue isn't related to Nvidia GPUs only.

**Comment by 0liu** [comment]
Same fat border issue with my i7-8700 with Intel UHD630. The problem is because of the display scaling set to 125% and it's gone when set to 100%. I have to scale my QHD 1440p display otherwise the fonts and icons and everything are too small. Is it possible to make it scale correctly? Thanks a lot! @vinceliuice

**Comment by tuschla** [comment]
> I have the same probems with alt version or without
> ![image](https://user-images.githubusercontent.com/8530497/96252009-921e0100-0fb1-11eb-88ac-255787c98379.png)
> 
> ![image](https://user-images.githubusercontent.com/8530497/96262565-3e67e380-0fc2-11eb-8aea-6e740c901c3b.png)

Exactly the same problem over here

**Comment by vinceliuice** [comment]
I uploaded a nvidia version for aurorae

**Comment by tuschla** [comment]
> I uploaded a nvidia version for aurorae

Thank you. It looks worse over here though. What am I doing wrong?
![image](https://user-images.githubusercontent.com/53692914/102092298-34c2f680-3e20-11eb-8eca-7a672d2015c5.png)

**Comment by 0liu** [comment]
> Same fat border issue with my i7-8700 with Intel UHD630. My two 1440 QHD monitors connect to DP and HDMI ports on the motherboard. The problem is because of the display scaling set to 125% and it's gone when set to 100%. I have to scale both displays otherwise the fonts and icons and everything are too small. Is it possible to make it scale correctly? Or will the problem be gone if switch to Wayland? Thanks a lot! @vinceliuice

I figured it out I have to choose the 1.25x one in Window Decoration. Sorry for false alarm.

**Comment by vinceliuice** [comment]
Delete the $HOME/.cache/kwin and all .kcache file then reboot you system

**Comment by tuschla** [comment]
Did not help unfortunately :(

**Comment by 0liu** [comment]
> Did not help unfortunately :(

After picking the 1.25x version Window Decoration, I installed my nvidia card tonight and the original theme still works. Maybe you should check you scaling too.

**Comment by tuschla** [comment]
> > Did not help unfortunately :(
> 
> After picking the 1.25x version Window Decoration, I installed my nvidia card tonight and the original theme still works. Maybe you should check you scaling too.

None of the scaled Window Decorations work.

**Comment by 0liu** [comment]
Actually the nvidia version makes the two upper corners broken. The original 1.25x works ok for me, but with nvidia card the window background became way too transparent.

![whitesur_nvidia](https://user-images.githubusercontent.com/4315932/102204002-26a3c300-3e7e-11eb-953f-d87dca8f457d.png)

#### Too transparent:

![too_transparent](https://user-images.githubusercontent.com/4315932/102204340-9914a300-3e7e-11eb-9113-c6305dcd9303.png)

**Comment by FunWithUrban** [comment]
I'm having all the problems mentioned above. I've tried every version of the window borders including the Nvidia one. This is a common problem with many other window border themes and not just yours. 

A few other examples of ones that don't work are Orchis and Materia and a few that do work are the official Breeze and Aritim. I do not have the experience to tell you why this is but maybe it can help you narrow things down. 

I'm using Kubuntu 20.10 with the latest Nvidia drivers on a 1440p 144hz gsync monitor for more added info.

**Comment by BrianRuizy** [comment]
> I'm using Kubuntu 20.10 with the latest Nvidia drivers on a 1440p 144hz gsync monitor for more added info.

+1, also using 1440p 144hz monitor, though when changing display settings the issue remains

**Comment by BrianRuizy** [comment]
@vinceliuice @n0x-io
I found a hack way of fixing this for those with Nvidia drivers. 
I edited the themes **rc** file (WhiteSur-darkrc)

``` zsh
cd ~/.local/share/aurorae/themes 
```
adjust left/right padding to your liking. 
```zsh
[Layout]
PaddingTop=35
PaddingBottom=80
PaddingRight=68
PaddingLeft=68
```

![image](https://user-images.githubusercontent.com/23439187/113183926-391dee80-921a-11eb-94fd-6b0bcfb50e03.png)
![image](https://user-images.githubusercontent.com/23439187/113184126-6cf91400-921a-11eb-8216-ee228a58d436.png)

**Comment by NadavKeren** [comment]
Did anyone find the issue? This seems to be a problem in other themes of @vinceliuice as well :(
I have also found the Nvidia version even worse (at least on Orchis).
